### PR TITLE
abl for post users - post client profile

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
@mjoyce91 i choose you to take over this PR 😭

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx [hunk 1]
```diff
@@ -191,7 +190,7 @@ const AvailableBidderTable = props => {
                           }
                           labelTextRight={
                             <Tooltip
-                              title="Bureau View"
+                              title="Bureau/Post View"
```
Comment (mjoyce91) [inline src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:193]: 👍

File: src/Constants/Menu.js [hunk 2]
```diff
@@ -246,15 +246,16 @@ export const GET_PROFILE_MENU = () => MenuConfig([
...
-      {
-        text: 'Available Bidders',
-        route: '/profile/post/availablebidders',
-        icon: 'users',
-        roles: [
-          'superuser',
-          'post_user',
-        ],
-      },
+      checkFlag('flags.available_bidders') ?
```
Comment (scott062) [inline src/Constants/Menu.js:249]: Do we want a flag for this?

Comment (elizabeth-jimenez) [reply src/Constants/Menu.js:249]: the others have a flag - so just being consistent

File: src/Containers/ProfilePublic/ProfilePublic.jsx [hunk 3]
```diff
@@ -29,6 +31,11 @@ class ProfilePublic extends Component {
     return isBureauView;
   }
 
+  isPostView = () => {
+    const viewType = get(this.props, 'match.params.viewType');
+    return viewType === 'post';
+  }
+
```
Comment (scott062) [inline src/Containers/ProfilePublic/ProfilePublic.jsx:38]: Not a showstopper but I'm thinking if there is really only two versions of ABL - CDA vs Non-CDA. Maybe a future story to revamp all these checks to simplify component prop checks. This goes for Client Profiles as well. Food for thought

**Review by scott062 — APPROVED** [review]
- [x] Post users see ABL menu
- [x] Post user sees same version of ABL as bureau
- [x] Post user sees limited client profiles
- [x] CDA sees new version of toggle text
- [x] Route includes /Post/
- [ ] Flag hides Post menu for ABL

**Pull request merged**